app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt

```diff
@@ -57,16 +56,14 @@ object IntentValidator {
 
     fun validate(context: Context, intent: SafeIntent): ValidatedIntentData? {
         setExperimentOverrides(intent, context)
-        changeFetchDelaySecondsForQA(intent)
+        changeFetchDelaySecondsForQA(intent, context)
 
         when (intent.action) {
             Intent.ACTION_MAIN -> {
-                val extras = intent.extras ?: return null
-
-                if (extras.keySet().contains(DIAL_PARAMS_KEY)) {
-                    val param = extras.getString(DIAL_PARAMS_KEY)
+                val dialParams = intent.extras?.getString(DIAL_PARAMS_KEY) ?: return null
+                if (dialParams.isNotEmpty()) {
                     TelemetryIntegration.INSTANCE.youtubeCastEvent()
-                    return ValidatedIntentData(url = "https://www.youtube.com/tv?$param", source = Session.Source.ACTION_VIEW)
+                    return ValidatedIntentData(url = "https://www.youtube.com/tv?$dialParams", source = Session.Source.ACTION_VIEW)
```

Comment:
It's probably not worth your time to fix now but this is still going to break blame: you'd need to edit the existing commit that originally changed this.